labcontrol/db/tests/test_process.py

```diff
@@ -1830,16 +1830,14 @@ def test_generate_sample_sheet_unrecognized_assay_type(self):
     def test_generate_amplicon_prep_information(self):
         # Sequencing run
         tester = SequencingProcess(1)
-        controls_sheet_id = tester.get_controls_prep_sheet_id()
         obs = tester.generate_prep_information()
-        exp = {1: EXPERIMENTAL_SAMPLES_PREP,
-               controls_sheet_id: CONTROL_SAMPLES_PREP_EXAMPLE}
+        exp_key = 'Test Run.1'
+        exp = {exp_key: COMBINED_SAMPLES_PREP_EXAMPLE}
         self.assertEqual(len(obs), len(exp))
         # print observed output, so it can be captured and compared
         # with expected string.
-        print(obs[1])
-        self.assertEqual(obs[1], exp[1])
-        self.assertEqual(obs[controls_sheet_id], exp[controls_sheet_id])
+        print(obs[exp_key])
```

Comment:
I think this is residual from testing :)

Reply:
Ok; it was pre-existing (see previous line 1840) so I didn't mess with it, but I am happy to take it out.